README.md

```diff
@@ -69,6 +69,7 @@ All pixels are parameterised by their component type.
 A trait that provides functions for manipulating images, parameterised over the image's pixel type.
 
 ```rust
+# use image::{Pixel, Pixels};
```

Comment:
A bit late, but should the `#` be here?

Reply:
Not necessarily. In the documentation this would of course make it invisible but for the readme this fails its purpose. I guess we can remove it if we want.